agent/pkg/models/models.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 type EntriesRequest struct {
-	LeftOff   int    `form:"leftOff" validate:"required,min=-1"`
+	LeftOff   string `form:"leftOff" validate:"required,min=-1"`
```

Comment:
min should be -1 after the change?

Reply:
I forgot to remove it. Removed with https://github.com/up9inc/mizu/commit/c2f5894de57c33d1aa6de1d4bdcda6fec3d62b4d